src/proto/ast.proto

```diff
@@ -67,6 +67,9 @@ message Declaration {
 	optional ChangeKind structural_change_kind = 10;
 	/** Kind of change on the label: UNCHANGED, ADDED, DELETED/REMOVED, RENAMED */
 	optional ChangeKind label_change_kind = 11;
+	/** For Specs use */
+	optional SpecStatement specs = 12;
```

Comment:
the field number for spec fields should start at 1000.

Reply:
Do we need to change the field number for these or only for the enums for specs?

Reply:
Done.